src/pages/evaluations/template.py

```diff
@@ -0,0 +1,181 @@
+            help="Name of the model family",
+            label_visibility="visible",
+            key="cross_model_comparison_model_family",
+        )
+
+    naive_logs = [
+        log
+        for log in naive_logs
+        if log.model_metadata.family == model_family or model_family is None
+    ]
+
+    # Get available metrics from filtered logs
+    task_metrics = set().union(*[get_metrics(log) for log in naive_logs])
+
+    with col4:
+        metric = st.selectbox(
+            "Metric",
+            sorted(task_metrics),
+            index=0,
+            help="Evaluation metric",
+            label_visibility="visible",
+            key="cross_model_comparison_metric",
+        )
+
+    if naive_logs:
+        # Display evaluation details from the first log entry
+        eval_details = naive_logs[0]
+        st.text("")  # Add a blank line for spacing
+        st.markdown("##### Evaluation details:")
+        # TODO: Add a link to task path
+        st.markdown(f"""
+                    - **Name:** {eval_details.eval_metadata.title}
+                    - **Description:** {eval_details.eval_metadata.description}
+                    - **ArXiv:** {eval_details.eval_metadata.arxiv}
+                    - **Task path in Inspect Evals:** [{eval_details.eval_metadata.path}](https://github.com/UKGovernmentBEIS/inspect_evals/tree/main/{eval_details.eval_metadata.path})
+                    """
+        )
+        st.text("")  # Add a blank line for spacing
+
+        fig_bar = create_bar_chart(naive_logs, metric)
+        st.plotly_chart(fig_bar)
+
+        fig_cutoff = create_cutoff_scatter(naive_logs, metric)
+        st.plotly_chart(fig_cutoff)
+
+        st.download_button(
+            label="Download chart data as JSON",
+            data=json.dumps([log.model_dump(mode="json") for log in naive_logs]),
+            file_name="eval_logs.json",
+            mime="application/json",
+        )
+
+    st.text("")  # Add a blank line for spacing
+    st.divider()
+    st.subheader("Pairwise analysis (unpaired)")
+    st.markdown("""
+                Here we compare two models directly by specifying one model as the baseline and the other as the test model across all evaluations in this group. We use their eval score and standard errors to test their difference for statistical significance. We highlight the cells where the confidence interval is in the positive or negative range signaling that the test model is statistically significantly better or worse compared to the baseline model.
+                """
+    )
+
+    col5, col6, col7 = st.columns(3)
+
+    with col5:
+        model_name = st.selectbox(
+            "Model name",
+            sorted(set(log.eval.model for log in eval_logs)),
+            index=0,
+            format_func=lambda option: option.split("/")[-1],
+            help="Name of the model to compare against",
+            label_visibility="visible",
+            key="pairwise_analysis_model_name",
+        )
```

Comment:
Oh man, this is __such a good feature__!! Thought: might want to be able to choose different types of statistical analyses.